play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializer.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.playjson
+
+import java.nio.charset.StandardCharsets
+
+import akka.actor.ExtendedActorSystem
+import akka.event.Logging
+import akka.serialization.SerializerWithStringManifest
+import play.api.libs.json._
+
+import scala.collection.immutable.Seq
+
+/**
+ * Internal API
+ *
+ * Akka serializer using the registered play-json serializers and provides
+ */
+final class PlayJsonSerializer(system: ExtendedActorSystem) extends SerializerWithStringManifest {
+
+  private val charset = StandardCharsets.UTF_8
+  private val log = Logging.getLogger(system, getClass)
+  private val isDebugEnabled = log.isDebugEnabled
+
+  private val registry = {
+    val registryClassName: String = system.settings.config.getString("lagom.serialization.play-json.serialization-registry")
+    system.dynamicAccess.createInstanceFor[SerializerRegistry](registryClassName, Seq.empty).get
+  }
```

Comment:
If we want access to it from more parts this must live somewhere else, could possibly be an Akka extension.